pkg/local_object_storage/shard/metrics_test.go

```diff
@@ -17,35 +17,36 @@ import (
 )
 
 type metricsStore struct {
-	objectCounters map[string]uint64
-	readOnly       bool
+	objCounters map[string]uint64
```

Comment:
Naming in one commit and renaming in another is not the best practice.

Reply:
fixed